src/Storage/Storage.Management/StorageAccount/StorageAccountBaseCmdlet.cs

```diff
@@ -80,14 +80,14 @@ protected struct AccountIdentityType
         {
             internal const string SystemAssigned = "SystemAssigned";
             internal const string UserAssigned = "UserAssigned";
-            internal const string SystemAssignedUserAssigned = "SystemAssignedUserAssigned";
+            internal const string SystemAssignedUserAssigned = "SystemAssigned, UserAssigned";
```

Comment:
The parameter value should not be changed, or it will be a breaking change, since script with original value will be broken.
Please handle it in PSH cmdlet internal code, instead change the user input parameter valid set.

Reply:
Fixed